manim/mobject/three_d/three_dimensions.py

```diff
@@ -358,8 +358,8 @@ def construct(self):
                     center=(3, 0, 0),
                     radius=1,
                     resolution=(20, 20),
-                    u_range=[0.001, PI - 0.001],
-                    v_range=[0, TAU]
+                    u_range=[0, TAU],
+                    v_range=[0, PI]
```

Comment:
The `u_range` and `v_range` default values are already `[0, TAU]` and `[0, PI]`, so we might as well simply remove these explicit arguments if the intention of the original example was to actually use `u` in `[0, TAU]` and `v` in `[0, PI]`.

The original example is, indeed, weird. Why did it use `u_range` and `v_range` in the first place? Maybe the idea could have been to show, for example, half a sphere by setting `v_range=[0, PI/2]`.

In order to showcase the proper use of `u_range` and `v_range`, may I ask you to please create a 3rd example where you use a different value for these parameters? Then, we could simply remove `u_range` and `v_range` from here.
```suggestion
```